### PR TITLE
Car demo dev

### PR DESCRIPTION
**Pull request by borditamas**

### CODE REVIEW

File: demo-car-consumer/src/main/java/com/aitia/demo/car_consumer/CarConsumerConstants.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package com.aitia.demo.car_consumer;
+
+public class CarConsumerConstants {
+	
+	//=================================================================================================
+	// members
+	
+	public static final String BASE_PACKAGE = "com.aitia";
```
Comment (rbocsi) [inline demo-car-consumer/src/main/java/com/aitia/demo/car_consumer/CarConsumerConstants.java]: it's ai.aitia :)

File: demo-car-consumer/src/main/java/eu/arrowhead/client/skeleton/consumer/security/ConsumerAccessControlFilter.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package eu.arrowhead.client.skeleton.consumer.security;
```
Comment (rbocsi) [inline demo-car-consumer/src/main/java/eu/arrowhead/client/skeleton/consumer/security/ConsumerAccessControlFilter.java]: I thought the consumer is not a web application so why we need an ACF class?

File: demo-car-provider/src/main/java/com/aitia/demo/car_provider/CarProviderConstants.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
+package com.aitia.demo.car_provider;
+
+public class CarProviderConstants {
+	
+	//=================================================================================================
+	// members
+	
+	public static final String BASE_PACKAGE = "com.aitia";
```
Comment (rbocsi) [inline demo-car-provider/src/main/java/com/aitia/demo/car_provider/CarProviderConstants.java]: ai.aitia

File: demo-car-provider/src/main/java/com/aitia/demo/car_provider/controller/CarServiceController.java [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+		final List<CarResponseDTO> response = new ArrayList<>();
+		for (final Car car : carDB.getAll()) {
+			boolean toAdd = true;
+			if (brand != null && !brand.isBlank() && !car.getBrand().equalsIgnoreCase(brand)) {
+				toAdd = false;
+			}
+			if (color != null && !color.isBlank() && !car.getColor().equalsIgnoreCase(color)) {
+				toAdd = false;
+			}
+			if(toAdd) {
```
Comment (rbocsi) [inline demo-car-provider/src/main/java/com/aitia/demo/car_provider/controller/CarServiceController.java]: missing space

File: demo-car-consumer/src/main/java/eu/arrowhead/client/skeleton/consumer/security/ConsumerAccessControlFilter.java (hunk 2 above)
Comment (borditamas) [reply demo-car-consumer/src/main/java/eu/arrowhead/client/skeleton/consumer/security/ConsumerAccessControlFilter.java]: It's 'ConditionalOnWebApplication' as it can be configured to act as web application.

File: client-skeleton-common/src/main/java/eu/arrowhead/client/skeleton/common/ArrowheadService.java [hunk 5]
```diff
@@ -0,0 +1,434 @@
+package eu.arrowhead.client.skeleton.common;
```
Comment (rbocsi) [inline client-skeleton-common/src/main/java/eu/arrowhead/client/skeleton/common/ArrowheadService.java]: why is this here? It should come from the client-library jar.
